src/Context.ts

```diff
@@ -35,7 +35,7 @@ export class Context {
     return this.req.roles ?? [];
   }
 
-  get log(): FastifyLoggerInstance {
+  get log(): FastifyBaseLogger {
```

Comment:
Is this the new version for it, should we change it in multiple places?

Reply:
`FastifyLoggerInstance` is deprecated. We should migrate, but didn't want to turn this PR into a migration 😬